frontend/views/containers/user-settings/Notification.vue

```diff
@@ -0,0 +1,30 @@
+<template lang='pug'>
+  .settings-container
+    .card
+      i18n.link(
+        tag='button'
+        @click.prevent='requestPermission'
+      ) Request Permission
+</template>
+
+<script>
+import { requestNotificationPermission } from '~/frontend/utils/notification.js'
+export default ({
+  name: 'NotificationSettings',
```

Comment:
This file should be renamed to `NotificationSettings.vue`

Reply:
Okay.